atmosphere/settings/__init__.py

```diff
@@ -198,7 +196,7 @@
 DEPLOY_SERVER_URL = SERVER_URL.replace("https", "http")
 
 # These DEFAULT variables can be overridden per provider..
-DEFAULT_NAMESERVERS = ['8.8.8.8','8.8.4.4.']
+DEFAULT_NAMESERVERS = ['8.8.8.8', '8.8.4.4.']
```

Comment:
The second IP seems wrong. Outside the scope of this PR fixing it, but keep it in mind.